package.json

```diff
@@ -8,11 +8,11 @@
     "url": "https://github.com/ajnart/homarr"
   },
   "scripts": {
-    "dev": "next dev",
+    "dev": "NODE_OPTIONS='--no-experimental-fetch' next dev",
     "build": "next build",
```

Comment:
It doesn’t apply to next build so won’t be exported in the container version

Reply:
Omg, makes sense. I'll adjust it

Reply:
Hehe